src/domains/dashboard/pages/Dashboard/Dashboard.tsx

```diff
@@ -23,6 +25,10 @@ export const Dashboard = ({ transactions, wallets, networks, portfolioPercentage
 	const activeProfile = useActiveProfile();
 	const history = useHistory();
 
+	React.useEffect(() => {
+		setScreenshotProtection(activeProfile?.settings().get(ProfileSetting.ScreenshotProtection) === true);
```

Comment:
```suggestion
		setScreenshotProtection(activeProfile?.settings().get(ProfileSetting.ScreenshotProtection) === false ? false : true);
```

This way screenshot protection will be enabled if `ProfileSetting.ScreenshotProtection` if falsy, but not `false`.

Reply:
fanks!